src/OpenTelemetry.Extensions.Hosting/Implementation/HostingExtensionsEventSource.cs

```diff
@@ -46,13 +46,13 @@ public void FailedOpenTelemetrySDK(Exception ex)
             }
         }
 
-        [Event(1, Message = "Failed to initialize: '{0}'. OpenTelemetry will not work.", Level = EventLevel.Error)]
+        [Event(1, Message = "An exception occured while adding OpenTelemetry Tracing to ServiceCollection. OpenTelemetry tracing will not work. Exception: '{0}'.", Level = EventLevel.Error)]
```

Comment:
Happy to get suggestion for better messaging here.
Here's what I intended to convey:
Something is wrong - the Exception is a hint to what might be the cause
The impact of that - tracing wont work.

Reply:
I kind of like the previous message better. I'm thinking maybe `Failed to initialize OpenTelemetry tracing. OpenTelemetry will not work. Exception: '{0}'`

Reply:
Its important to clearly state what will not work  - OpenTelemetry tracing is what is broken here. We don't know if metrics/logs work. So stating only what this code knows - "OTel tracing will not work" instead of "OTel will not work".
thoughts?

Of the following message, the _italicised_ portion is not very clear I believe.. Will wait to get more opinions on this.
"An exception occured _while adding OpenTelemetry Tracing to ServiceCollection_. OpenTelemetry tracing will not work. Exception: '{0}'."

Reply:
I just personally think the ServiceCollection part is kind of irrelevant and that this is an error that occurred while attempting to initialize. So maybe `An exception occured while initializing OpenTelemetry Tracing. OpenTelemetry tracing will not work. Exception: '{0}'.`?

Reply:
Got it. This sounds better.
An exception occurred while initializing OpenTelemetry tracing. OpenTelemetry tracing will not work. Exception: '{0}'.

Keeping conversation open to hear more thoughts.

Reply:
Instead of "will not work" how about "will remain disabled"? Or maybe "will be non-functional"?